### PR TITLE
IBigtableDataClient interface - implement mutateRows method

### PR DESCRIPTION
**Pull request by elisheva-qlogic**

### CODE REVIEW

File: bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataClientWrapper.java [hunk 1]
```diff
@@ -57,6 +58,15 @@ public void mutateRow(RowMutation rowMutation) {
     throw new UnsupportedOperationException("Not implemented yet");
   }
 
+  public void mutateRows(RowMutation rowMutation) {
```
[the remark this answers is not in the document]
Comment (elisheva-qlogic) [reply bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataClientWrapper.java]: Because it isn't overriding any method from its super class

[the remark this answers is not in the document]
Comment (sduskis) [reply bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataClientWrapper.java]: I guess I don't understand why you're adding this method.  If this method ought to be used, then we need a new method in the interface.

File: bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/AbstractBigtableTable.java [hunk 2]
```diff
@@ -356,8 +356,8 @@ public ResultScanner getScanner(byte[] family, byte[] qualifier) throws IOExcept
   @Override
   public void put(Put put) throws IOException {
     LOG.trace("put(Put)");
-    RowMutation request = hbaseAdapter.adapt(put);
-    mutateRow(put, request, "put");
+    RowMutation rowMutation = hbaseAdapter.adapt(put);
```
Comment (sduskis) [inline bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/AbstractBigtableTable.java]: Please make this change in a separate PR.  There's no reason to lump this change with `mutateRows`
